modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -1,15 +1,11 @@
 :_module-type: PROCEDURE
 
 [id="deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_{context}"]
-= Deploying models on single node {openshift-platform} using KServe raw deployment mode
+= Preparing your cluster for standard mode deployment
```

Comment:
IMO, this whole file should be removed, and its contents should be moved to Chapter 5 of the installation manual: https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.18/html/installing_and_uninstalling_openshift_ai_self-managed/installing-the-single-model-serving-platform_component-install#configuring-automated-installation-of-kserve_component-install

Reply:
> Should we remove any content from here that shows how to install your cluster for raw deployment mode? I think it's confusing to have a topic that repeats installation content.

I agree. We should remove any content related to installation.